Module4_Labs/Lab4_Maze_Solver_With_Stack/Cards/131.md

```diff
@@ -10,10 +10,29 @@ Define `isFull` as :
 return ((self.index + 1) == ROWS*COLUMNS if True else False)
 ```
 
+It's the same as doing the following
```

Comment:
For this card, the code could use a little more explanation. Right now, the card is mainly just code